pkg/infra/teams/team1/team1_agent.go

```diff
@@ -404,7 +404,7 @@ func (s *SocialAgent) UpdateSelfishness(agent agent.BaseAgent) {
 	}
 }
 
-func (s *SocialAgent) UpdateInternalState(self agent.BaseAgent, fightResult *commons.ImmutableList[decision.ImmutableFightResult], _ *immutable.Map[decision.Intent, uint], _ chan<- logging.AgentLog) {
```

Comment:
Maybe revert that one, since its in another teams folder?

Reply:
I think those were just changes when i merged main on our branch so they should be ok?

Reply:
Interesting... i can definitely see those changes on the main branch but should't appear here since the comparison is between your branch and the updated main version...

Reply:
yeah, seemed weird to me as well.